setup.py

```diff
@@ -42,7 +42,8 @@ def run_tests(self):
     license='Expat license',
     package_dir={'': 'src'},
     py_modules=['pep8ext_naming'],
-    install_requires=['flake8>=3.9.1'],
+    install_requires=['flake8>=5.0.0'],
+    extras_require={"dev": ["tox"]},
```

Comment:
I'm not sure we want this. We _do_ use tox to run our full text matrix, but tox isn't explicitly needed to develop this package. It's optional metadata, but I don't think it's particularly helpful unless we tell people about it in e.g. the `README.md` or a new `CONTRIBUTING.md` file.

... so let's leave this out for this change.

Reply:
My "nominal" project structure goes:

```
$ cat requirements.txt:
...
$ cat requirements-dev.txt:
-r requirements.txt

...
```

So that people can start "full-stack" working/developing with a simple venv+`pip install -r requirements-dev.txt`.

I understand that my opinions/workflows "do not reflect to everyone", so I will revert also this change.
Low-friction setup time though is low-friction/"replicatable" contribution 🙏

Reply:
I understand. My main point is that it strays outside the bounds of this particular change and would be more appropriate in a separate proposal that also updates the documentation, etc.